pkg/services/ngalert/api/tooling/definitions/alertmanager.go

```diff
@@ -81,9 +81,10 @@ import (
 //     Responses:
 //
 //       200: Ack
-//       207: MultiStatus
```

Comment:
I think we want to keep Lines 84 and 96 don't we as the API can return both of these HTTP status codes?

Reply:
I've checked but I didn't find where we return those statuses. Can you point me there? But I agree that in case we do, we have to keep them here.

Reply:
my bad; I have put them back! can you please check again?